core/comment.js

```diff
@@ -26,6 +26,7 @@
 
 goog.provide('Blockly.Comment');
 
+goog.require('Blockly.Block');
```

Comment:
There's a circular dependency here that creates a compiler error:
Blockly.Comment requires Blockly.Block, and Blockly.Block requires Blockly.Comment..